src/Menu/MenuItem.js

```diff
@@ -53,13 +66,31 @@ class MenuItem extends React.PureComponent {
         }}
       >
         <input {...{ id, name, value, css: hideVisually(), type: 'hidden' }} />
-        <Icon
-          name="symbol-circle"
-          mr="small"
-          color={baseColor}
-          fontSize="16px"
-        />
-        <Text.span fontSize="size4">{label}</Text.span>
+        <Flex width="100%" justifyContent="space-between">
+          <Flex
+            flex="1 0 auto"
+            maxWidth={
+              secondaryLabel ? `calc(100% - ${MIN_WIDTH} - ${PADDING})` : '100%'
```

Comment:
I could not, for the life of me, make this work without `calc`. AFAIK, all browsers we support can handle this.